e2e-tests/scaling/run

```diff
@@ -52,6 +52,7 @@ kubectl_bin get psmdb -o yaml ${cluster%%-rs0} \
     | kubectl_bin apply -f-
 
 desc 'check if Pod deleted'
+sleep 20
```

Comment:
1. what is the cause of 20 extra seconds wait?
2. consider changing wait_for_delete instead of sleep 20

the same relates to https://github.com/percona/percona-server-mongodb-operator/pull/364/files#diff-d4eaa2979fe73f78cc5d66c142d3d1a7R68